clearwater-socket-factory/clearwater-socket-factory-mgmt.conf

```diff
@@ -0,0 +1,23 @@
+# clearwater-socket-factory-mgmt
+#
+# This process runs as root in the management namespace, and enables processes
+# running in other namespaces to obtain sockets in this namespace.
+
+description "Clearwater management socket factory server"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+
+script
+  WHITELIST_DIR=/etc/clearwater-socket-factory/management.d
+  NAMESPACE=management
+
+  . /usr/share/clearwater/bin/clearwater-socket-factory-common
+
+  # Sleep for 5 seconds, to avoid upstart respawning socket-factory too quickly
+  sleep 5
+
+  /usr/share/clearwater/bin/clearwater_socket_factory $DAEMON_ARGS
```

Comment:
It was because of the sleep, (which we don't want in CentOS), but I suppose theres no reason why it has to happen in the middle of the processing (and you suggest an alternative to the sleep anyway).  I'll refactor